### ISSUE TITLE
@ababinchak this sounds like an opportunity for pull requests. Can you help?

### ISSUE THREAD

**Issue by TimmyTurner001** [devcontainers/images#987]
@ababinchak this sounds like an opportunity for pull requests. Can you help?

_Originally posted by @banreyms in https://github.com/microsoft/Security-101/issues/2#issuecomment-1964309164_

**Comment by samruddhikhandale** [comment]
> I would strongly recommend that URL's be coded to open in a new tab. Otherwise, student will have difficulty getting back to the course. The URL resource at the bottom on the page does not open in a new tab

I am not sure if I understand the context here. Which IDE are we talking about, how is this relevant to the devcontainer/images?

**Comment by bamurtaugh** [comment]
I don't think this is related to dev containers, but please let us know if it is, and I'm happy to reopen. Thanks!
